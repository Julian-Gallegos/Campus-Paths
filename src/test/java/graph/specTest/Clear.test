# Tests that all elements in a graph can be cleared

# Create an empty graph
CreateEmptyGraph graph1

# Add a pair of nodes
AddNode graph1 node1
AddNode graph1 node2

# Add an edge from node1 to node 2
AddEdge graph1 node1 node2 edge1

# Remove all nodes and edges in graph1
Clear graph1