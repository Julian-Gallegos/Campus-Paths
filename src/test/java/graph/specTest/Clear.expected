# Tests that all elements in a graph can be cleared

#Create an empty graph
created graph graph1

# Add node to a graph
added node node1 to graph1
added node node2 to graph1

# Add an edge from node1 to node 2
added edge edge1 from node1 to node2 in graph1

# Remove all nodes and edges in graph1
All elements in graph1 cleared